.github/workflows/sbom.yml

```
@@ -0,0 +1,43 @@
+    - uses: actions/checkout@v4
+    
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.11'
+    
+    - name: Generate SBOM
+      run: python scripts/generate_sbom.py
+    
+    - name: Upload to Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)